docs/intro/config.md

```diff
@@ -33,8 +33,9 @@ documents: ./client/src/graphql/**/*.graphql
 extensions:
   # Graphback configuration
   graphback:
-    ##  Location to file that is contains GraphQL schema that will be processed by Graphback
-    model: ./model
+    ##  Path to file that contains GraphQL schema that will be processed by Graphback
+    ## Can be an array of paths too
+    model: ./model/**/*.graphql
```

Comment:
@craicoverflow is this good enough or should I add some actual sentence to the doc explaining it?

Reply:
It is self explanatory for me

Reply:
I think it is self explanatory :)